### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SeanCode15**

### CODE REVIEW

**Review by seanboltjes — COMMENTED** [review]
@Kboltjes  check this

**Comment by Kboltjes** [comment]
- [ ] You gotta do something about this.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # utils
-A place for all the utility functions and helper functions. Most of this are pretty broad in the scope that they can be used in. Import these into your projects to have standardized utils
+A place for all the utility functions and helper functions. Most of this are pretty broad in the scope that they can be used in. Import these into your projects to have standardized utility functions
```
Comment (Kboltjes) [inline README.md]: - [x] Period needed at end of sentence.

**Review by Kboltjes — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SeanCode15) [reply README.md]: check now

**Review by Kboltjes — APPROVED** [review]

**Pull request merged**